samples/basic-usage/ui5.yaml

```diff
@@ -10,6 +10,8 @@ server:
               service:
                   urlBasePath: /here/goes/your/serviceurl
                   name: ''
+                  # TODO: old variable names, rename them to the new ones
                   metadataXmlPath: ./webapp/localService/metadata.xml
+                  # TODO: is mockdataPath necessary when folder not defined?
```

Comment:
it's necessary as soon as you want mockdata to be found :)

Reply:
Ok, it may makes sense to add the folder including the config in the yaml to create the structure when you may add mockdata later